elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -35,26 +35,60 @@ export interface P2PPenalty {
 	readonly penalty: number;
 }
 
-export interface P2PPeerInfo {
+export interface P2PSharedState {
 	readonly ipAddress: string;
 	readonly wsPort: number;
 	// tslint:disable-next-line: no-mixed-interface
 	readonly [key: string]: unknown;
 }
 
+export enum ConnectionKind {
+	OUTBOUND = 'outbound',
```

Comment:
If you introduce this, we must remove `PEER_KIND_OUTBOUND` and `PEER_KIND_INBOUND` from `constants.ts`.